packages/mc-scripts/src/utils/auth.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+
+const authenticator = async (email, password, mcApiUrl) => {
+  const response = await fetch(`${mcApiUrl}/tokens`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ email, password }),
+  });
+
+  if (response.status !== 200) {
+    const res = await response.json();
```

Comment:
Nit: note that attempting to parse JSON is not always safe. We can't assume that the response body is always JSON.

Usually we have a function `processResponse` where we resolve the response using `response.text()` and attempt to safely parse it to JSON.